packages/subscription-manager/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-restricted-imports */
+/**
+ * External dependencies
+ */
+import { ReactNode } from 'react';
+import SectionNav from 'calypso/components/section-nav';
+import SectionNavTabs from 'calypso/components/section-nav/tabs';
+import './styles.scss';
+
+type TabsProps = {
+	children: ReactNode;
+};
+
+const Tabs = ( { children }: TabsProps ) => (
+	<SectionNav className="subscription-manager-tabs">
+		<SectionNavTabs>{ children }</SectionNavTabs>
+	</SectionNav>
```

Comment:
This seems to be a straightforward wrapper that doesn't add anything but an extra class name. Seems like it could be avoided in favor of using the original components, no?